nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -150,7 +150,7 @@ in {
           export XDG_DATA_DIRS=$XDG_DATA_DIRS''${XDG_DATA_DIRS:+:}${mimeAppsList}/share
 
           # Override gsettings-desktop-schema
-          export XDG_DATA_DIRS=${nixos-gsettings-desktop-schemas}/share/gsettings-schemas/nixos-gsettings-overrides''${XDG_DATA_DIRS:+:}$XDG_DATA_DIRS
+          export NIX_GSETTINGS_OVERRIDES_DIR=${nixos-gsettings-desktop-schemas}/share/gsettings-schemas/nixos-gsettings-overrides/glib-2.0/schemas
```

Comment:
Could there be a use case where `$NIX_GSETTINGS_OVERRIDES_DIR` was set already before this command?

Reply:
Anyway, that can be amended later without a larger rebuild.